swagger_to_sdk_config.schema.json

```diff
@@ -0,0 +1,418 @@
+            }
+          },
+          "patternProperties": {
+            "sdkrel:.*": {
+              "$ref": "#/definitions/sdkrelProperty"
+            }
+          },
+          "description": ""
+        },
+        "wrapper_filesOrDirs": {
+          "$ref": "#/definitions/wrapper_filesOrDirs"
+        },
+        "delete_filesOrDirs": {
+          "$ref": "#/definitions/delete_filesOrDirs"
+        },
+        "generated_relative_base_directory": {
+          "$ref": "#/definitions/generated_relative_base_directory"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "version"
+      ]
+    },
+    "projects": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "object",
+        "properties": {
+          "markdown": {
+            "type": "string",
+            "minLength": 1,
+            "description": "This is an optional parameter which specificy the Autorest MD file path for this project. This is relative to the rest-folder paramter."
+          },
+          "autorest_options": {
+            "$ref": "#/definitions/autorest_options"
+          },
+          "wrapper_filesOrDirs": {
+            "$ref": "#/definitions/wrapper_filesOrDirs"
+          },
+          "delete_filesOrDirs": {
+            "$ref": "#/definitions/delete_filesOrDirs"
+          },
+          "generated_relative_base_directory": {
+            "$ref": "#/definitions/generated_relative_base_directory"
+          },
+          "output_dir": {
+            "type": "string",
+            "minLength": 1,
+            "description": "This is the folder in your SDK repository where you want to put the generated files."
+          },
+          "build_dir": {
+            "type": "string",
+            "minLength": 1,
+            "description": "This is an optional folder where to put metadata about the generation (Autorest version, date of generation, etc.). This can be used by our monitoring system to detect package that needs an update. Be sure this folder is unique in the entire file, to avoid overwritting a file from another project."
+          }
+        },
+        "additionalProperties": false,
+        "description": ""
+      },
+      "description": "It's a dict where keys are a project id. The project id has no constraint, but it's recommended to use namespace style, like \"datalake.store.account\" to provide the best flexibility for the --project parameter."
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "meta"
+  ],
+  "definitions": {
+    "autorest_options": {
+      "properties": {
+        "verbose": {
+          "enum": [
+            "",
+            true,
+            false
+          ],
+          "description": "An option that will be passed to autorest.\nShow verbose output information."
```

Comment:
Is 

> An option that will be passed to autorest

necessary if it is present in all descriptions in this block and block is names `autorest_options`?

Reply:
No, it's not. I'll remove that. Thanks for finding it.

Reply:
Oh, wait. This is in the schema. I left that there because this was just a copy and paste from the AutoRest documentation, and it seemed okay to leave it.